docs/usage/nuget.md

```diff
@@ -45,17 +56,21 @@ Alternative feeds can be specified either [in a `NuGet.config` file](https://doc
 }
 ```
 
-In this example we defined 3 NuGet feeds.
-The package resolving process uses the `merge` strategy to handle the 3 feeds.
+In the example above we've set three NuGet feeds.
+The package resolving process uses the `merge` strategy to handle the three feeds.
 All feeds are checked for dependency updates, and duplicate updates are merged/joined together into a single dependency update.
 
 If your project uses lockfiles (a `package.lock.json` exists), alternate feed settings must be defined in a `NuGet.config` only, as `registryUrls` are not passed through to the NuGet commands used.
 
 ### Protocol versions
 
-NuGet supports two protocol versions, `v2` and `v3`, the NuGet client and server must use the same protocol version.
-Renovate as a NuGet client supports both versions and will use `v2` unless the configured feed URL ends with `index.json` (which mirrors the behavior of the official NuGet client).
-If you have a `v3` feed that does not match this pattern (e.g. JFrog Artifactory) you need to help Renovate by appending `#protocolVersion=3` to the registry URL:
+NuGet supports two protocol versions, `v2` and `v3`.
+The NuGet client and server must use the same version.
+
+Renovate as a NuGet client supports both `v2` and `v3` protocols, and will use `v2` unless the configured feed URL ends with `index.json`.
+This mirrors the behavior of the official NuGet client.
+
+If you have a `v3` feed that doesn't match this pattern (e.g. JFrog Artifactory) you need to help Renovate by appending `#protocolVersion=3` to the registry URL:
```

Comment:
We have text about things not matching a pattern, but we don't explain what the pattern is in the first place. Do you maintainers have a good example we can use here? It feels like some important stuff is left out of our text right now.

> If you have a `v3` feed that doesn't match this pattern (e.g. JFrog Artifactory) you need to help Renovate by appending `#protocolVersion=3` to the registry URL: